### ISSUE TITLE
A different approach to tests?

### ISSUE THREAD

**Issue by koddsson** [kristjanmik/apis#154]
You may have noticed I've been trying to get travis to run the tests consistently as green which has mostly been me adding timeout limits to endpoints that take longer than others and skipping tests that error out.

The former is kind of hard since in the travis CI we don't utilize the redis cache layer. I've been pondering this and I'm wondering if we could up the timeout of all the tests to something really high? My thinking is that we want to test the "correctness" of the service and not the speed of it with the integration tests.

tl;dr What do you guys think about setting the global timeout for tests to something really high (like 20sec) to get the tests running moar green?

**Comment by koddsson** [comment]
cc/ @MiniGod @kristjanmik @benediktvaldez

**Comment by rthor** [comment]
:+1:

**Comment by kristjanmik** [comment]
I agree. We should just bump the global timeout. The only downside is when you are running the tests locally it can take some time to complete...

**Comment by koddsson** [comment]
I have a PR with this change at #156 which does this and it seems to work fine. It even seems to outperform the previous runs by a small number on travis.

![screen shot 2015-02-02 at 23 08 32](https://cloud.githubusercontent.com/assets/318208/6011102/462ea470-ab30-11e4-9e8d-257fa64ce9a3.png)

Maybe the error handling takes a lot of time or something :confused:

**Comment by kristjanmik** [comment]
The timeout does not matter that much unless it is a miss in which case the test fails. Then each miss will add 20secs to the test. 
Getting the tests green is a higher priority though, I'm merging #156
